deployment/conf/.templates/start_server.sh.templ

```diff
@@ -7,6 +7,8 @@ if [ "$POOL_PASSWORD" ] ; then
     /usr/sbin/condor_store_cred -p "$POOL_PASSWORD" -f `condor_config_val SEC_PASSWORD_FILE`
 fi
 
+chown kbase `condor_config_val SEC_PASSWORD_FILE`
```

Comment:
Are the daemons running as root or as kbase?  I thought we switched to root.

Reply:
This is not for the daemons. This is for starting the java server. However, the KBASE user needs to be able to read that password file.

Reply:
@kkellerlbl